client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -32,7 +32,7 @@ class NameserversToggle extends React.PureComponent {
 				</span>
 
 				<form className="name-servers__toggle">
-					<FormToggle
```

Comment:
I think we should open an a11y issue for this toggle as well, right? It has no corresponding label that points to it's ID.

Reply:
Good call - opened #53003 to keep track of that one.